project/webapp/templates/partials/account/utils/representative_card.html

```diff
@@ -10,6 +10,12 @@
                 <div class="text-lato">Party: {{party}}</div>
             </div>
             <div class="subtitle-lato alignment">-</div>
+            <div class="section">
+                <div class="link-lato name"> Votes opposed like me: {{ votes.opposed_the_same }}</div>
```

Comment:
I think these should be "boiled down" to "voted same" and "voted different" counts.

Reply:
🆗